_layouts/default.html

```diff
@@ -32,5 +35,12 @@
         {% include footer.html %}
         </div>
       </div>
+        
+        
+    <script>
```

Comment:
***my suggestion is that we create a separate javascript file with the script and link it to this file. please tell me what you think. looking forward. T.***